models/lfs_lock.go

```diff
@@ -22,7 +23,6 @@ type LFSLock struct {
 	ID      int64       `xorm:"pk autoincr"`
 	Repo    *Repository `xorm:"-"`
 	RepoID  int64       `xorm:"INDEX NOT NULL"`
-	Owner   *User       `xorm:"-"`
```

Comment:
Is the linter able to detect unused members?

Reply:
The linter is able to detect unused private members, but exported one like these aren't detected.

I think the linter is preventing to detect this, as this Field could be used when go-gitea/gitea is imported as package and a golang program is using that field. But given that go-gitea/gitea shouldn't be used as package(right?), we can modify the deadcode linter and be more aggressive and also check on these Public fields etc. Not sure which refactor PR it was mentioned, but we also could check if a certain function is only used with the `xxx_test.go` packages and move it to them.

https://github.com/golangci/go-misc/blob/master/deadcode/deadcode.go

Reply:
Currently we expect no one imports gitea. Otherwise this change would not be possible as we break the "api".

Reply:
Got it, I will open up a issue for such proposal. Should I open up a issue at [gitea-vet](https://gitea.com/gitea/gitea-vet/issues) or just here.